File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/ordering/services/OrderRepository.java
package com.metaengine.demo.ordering.services;

import java.util.List;

import com.metaengine.demo.ordering.aggregates.Order;

/** OrderRepository persistence service. */
public class OrderRepository {

    /** Persist an Order from a partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id (may return null). */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List orders up to limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-invocation-script/run-001/b-baseline/output/src/domain/ordering/services/OrderRepository.ts
import { Order } from '../aggregates/Order';

/** OrderRepository service. */
export class OrderRepository {
  create(input: Partial<Order>): Order {
    void input;
    throw new Error("not implemented");
  }
  findById(id: string): Order | null {
    void id;
    throw new Error("not implemented");
  }
  list(limit: number): Order[] {
    void limit;
    throw new Error("not implemented");
  }
  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}
